test/integration/features/cmd-config.feature

```diff
@@ -174,10 +174,17 @@ user defined options which changes default behaviour of Minishift.
     | property        | value                                                                                            | expected                                                                                        |
     | iso-url         | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso        | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso       |
     | iso-url         | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso  | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso |
-    | iso-url         | file:///home/me/Downloads/my_handmade_b2d.iso                                                    | file:///home/me/Downloads/my_handmade_b2d.iso                                                   |
     | iso-url         | b2d                                                                                              | b2d                                                                                             |
```

Comment:
Remove  the row

Reply:
The `iso-url` with file scheme is already removed.

Reply:
Right, but i am talking about this
| iso-url         | b2d                                                                                              | b2d                                                                                             |

Reply:
Same as above.